assignment7.txt

```diff
@@ -0,0 +1,16 @@
+1)
+    ++*p increments the value that p is pointing to
+    *p++ gets the value that p is pointing to, then increments p
+    *++p increments p, then returns the new value that p is pointing at
+2) yes?
+3) You can pass by reference
+4.1) char *
+4.2) 121
```

Comment:
"xyz" is just an array of characters, so the "xyz"[1] is just accessing "y". Then you just subtract 'y' from it to get 0. This is because, if you can recall, a char is just a value mapped to a character!